idea/src/org/jetbrains/kotlin/idea/intentions/conventionNameCalls/ReplaceContainsIntention.kt

```diff
@@ -48,7 +48,8 @@ class ReplaceContainsIntention : SelfTargetingRangeIntention<KtDotQualifiedExpre
         if (!element.isReceiverExpressionWithValue()) return null
 
         val functionDescriptor = getFunctionDescriptor(element) ?: return null
-        if (!functionDescriptor.isOperator || !OperatorChecks.check(functionDescriptor).isSuccess) return null
+
+        if (!functionDescriptor.isOperator) return null
```

Comment:
I removed this test.
https://github.com/takahirom/kotlin/blob/145ee2c92f909706272922e2f3eab03d64d853ec/idea/testData/intentions/conventionNameCalls/replaceContains/withoutOperatorModifier.kt
Please check .